### PR TITLE
Mark AutoRelockTime attribute of the Door Lock cluster as optional in the XML

### PR DESCRIPTION
**Pull request by Thirsrin**
Description:

AutoRelockTime attribute is optional attribute for doorlock cluster(doorlock.xml) as per matter specs(sec 5.2.9).

### CODE REVIEW

**Comment by bzbarsky-apple** [comment]
So this is not doing a "sync" with the spec like the PR subject says, it's just marking the one attribute as optional?

File: src/app/zap-templates/zcl/data-model/chip/door-lock-cluster.xml
Comment (bzbarsky-apple) [inline src/app/zap-templates/zcl/data-model/chip/door-lock-cluster.xml:1]: The change here is fine, but the commit message needs to be fixed to describe what's actually changing.  This is not a general "make door lock XML follow the spec" change, yes?

**Review by jmartinez-silabs — APPROVED** [review]

**Review by bzbarsky-apple — APPROVED** [review]

**Comment by andy31415** [comment]
@Thirsrin this is a backwards-incompatible change (API change). may need a bit more review if/why this would be safe.

**Review by andy31415 — DISMISSED** [review]
Marking this as hold a bit, until we are able to discuss (likely offline from this PR) about sideffects of API changes.

What we wish we had is full API stability, however in practice this is hard if the spec says one thing and the SDK implements something else.

Looking to figure out what to do here.

**Pull request merged**